src/actions/topicActions.js

```diff
@@ -0,0 +1,15 @@
+import * as types from './actionTypes';
+import topicApi from '../api/topicApi';
+
+export const getTopicList = (topics) => ({
+  type: types.GET_TOPICS,
+  topics
+});
+
+export const getTopics = () => (dispatch) => {
+  return topicApi.topics().then(response => {
+    dispatch(getTopicList(response));
```

Comment:
I'd prefer to send the value `response.topics` since I saw a unnecessary double prop `topics` here: https://github.com/fedeagripa/TargetMVD_React/pull/4/files#diff-26e6e7539428d3090fbe986d8b5eb401R7